src/openrct2/ride/CableLift.cpp

```diff
@@ -39,7 +39,7 @@ rct_vehicle* cable_lift_segment_create(
         move_sprite_to_list((rct_sprite*)current, SPRITE_LIST_TRAIN * 2);
         ride->cable_lift = current->sprite_index;
     }
-    current->is_child = head ? 0 : 1;
+    current->type = head ? 0 : 1;
```

Comment:
It would be better to use an enum for `type`.

Reply:
How can I do that when type is different for each sprite identifier?

Reply:
I meant to say "define an enum with constants for 0 and 1, and leave type as an uint8_t".

Reply:
done